go.mod

```diff
@@ -60,8 +59,8 @@ require (
 	github.com/magefile/mage v1.15.0
 	github.com/mattn/go-colorable v0.1.13
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/remotedialer v0.4.5-rc.2
-	github.com/rancher/system-agent v0.3.13-rc.3
+	github.com/rancher/remotedialer v0.5.0
```

Comment:
Just want to call out that the `remotedialer` dependency is not used in the current provisioned flow. This is only used when defining proxies via wins, which is an RKE1 specific behavior that is no longer used in any capacity.